utils/build-script

```diff
@@ -162,7 +162,7 @@ Using option presets:
   using the name=value syntax on the command line.
 
 
-Any arguments passed after "--" are forwarded directly to Swift's
```

Comment:
I'm afraid we have to support passing arguments after `--`, many systems I know of depend on this behavior.